engine/apps/heartbeat/models.py

```diff
@@ -43,9 +42,14 @@ class IntegrationHeartBeat(models.Model):
 
     created_at = models.DateTimeField(auto_now_add=True)
     timeout_seconds = models.IntegerField(default=0)
+
+    # Stores the latest received heartbeat signal time
     last_heartbeat_time = models.DateTimeField(default=None, null=True)
```

Comment:
small tip. If you include comments _below_ a class attribute, your IDE can nicely show you that comment when using this attribute:

```suggestion
    last_heartbeat_time = models.DateTimeField(default=None, null=True)
    """
    Stores the latest received heartbeat signal time
    """
```

<img width="551" alt="Screenshot 2023-08-02 at 09 43 54" src="https://github.com/grafana/oncall/assets/9406895/266a0990-648d-4fda-99c2-241f9891aee6">